clients/java/zms/src/main/java/com/yahoo/athenz/zms/ZMSClient.java

```diff
@@ -1064,20 +1064,36 @@ public Role getRole(String domainName, String roleName, boolean auditLog, boolea
      * @param domainName name of the domain
      * @param roleName   name of the role
      * @param auditRef   string containing audit specification or ticket number
+     * @param returnObj Boolean returns the updated object from the database if true
      * @param role       role object to be added to the domain
      * @throws ZMSClientException in case of failure
      */
-    public void putRole(String domainName, String roleName, String auditRef, Role role) {
+    public void putRole(String domainName, String roleName, String auditRef, Boolean returnObj, Role role) {
```

Comment:
So if the return value is still void how would we get the object back in Java? Shouldn't this return the object back to the client which would be either null if the retrunObj is false or the actual updated object?

same comment applies to all new api calls